Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -58,6 +58,8 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: ViewController,
         
         controller.setDelegate(self)
         controller.synchronize()
+        
+        navigationItem.backButtonTitle = ""
```

Comment:
I just realized `ChatChannelListVC` doesn't use the lifecycle methods. We'll fix that later...